src/prefect/tasks/prefect/flow_run.py

```diff
@@ -29,7 +30,9 @@ class StartFlowRun(Task):
             task will wait until the flow run is complete, and then reflect the corresponding
             state as the state of this task.  Defaults to `False`.
         - scheduled_start_time (datetime, optional): the time to schedule the execution
-                for; if not provided, defaults to now
+            for; if not provided, defaults to now
+        - create_link_artifact (bool, optional): create a link artifact that links to the
+            created flow run page in the UI. Defaults to `True`.
```

Comment:
Is there a reason a user would ever set this to False? If not, we might skip the option entirely until its requested.

Reply:
Yeah that's fair, no real need for it until requested